Maestro/subscriptions.json

```diff
@@ -942,8 +942,6 @@
     {
       "triggerPaths": [
         "https://github.com/dotnet/versions/blob/master/build-info/dotnet/standard/release/2.0.0/Latest.txt",
-        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/2.0.0/Latest.txt",
-        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/core-setup/release/2.0.0/Latest.txt",
```

Comment:
Keep in mind these only affect the frequency of updates (by triggering them), and the update PR contents depend only on dependencies.props. E.g. `RemoteDependencyBuildInfo Include="CoreFx"` and `XmlUpdateStep Include="CoreFx"` and the Core-Setup ones should be disabled/commented/removed from https://github.com/dotnet/wcf/blob/release/2.0.0/dependencies.props to match this trigger change.

The same goes for the others--like in WCF `master`, you'll have to update dependencies.props to point to `release/2.1` build-infos.

Reply:
I think I'd actually suggest doing the WCF PRs first, since that describes what will happen during upgrades, then coming back to make the triggers match.

Reply:
Yup, I'm going through the dependencies.props files in each of our branches to do what we want. Since I had to update at least the trigger paths for our Master branch I thought I'd "cleanup" all the entries for WCF.